contracts/Strategy.sol

```diff
@@ -234,40 +218,54 @@ contract Strategy is BaseStrategy {
         return
             string(
                 abi.encodePacked(
-                    "SingleSidedCrv",
-                    IERC20Extended(address(want)).symbol()
+                    "SingleSidedCrvSynth",
+                    IERC20Extended(address(_synthCoin())).symbol()
```

Comment:
any reason why you didn't use the name variable we used in the other strats?

Reply:
we have enough space in the bytecode. when possible, I like automating this to reduce points of failure